ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -140,6 +149,9 @@ message CreateUserRequest {
     // which SHOULD include appropriate rights for the ``user.primary_party``.
     // Required
     repeated Right rights = 2;
+
+    // Optional
+    repeated Property properties = 3;
```

Comment:
With a fixed limit, e.g. max 10 properties per user

Reply:
Definitely agree this needs a limit. 10 feels perhaps low to me, but certainly easier to start restrictive and raise the limit if we find that we have to :+1: